netlify.toml

```diff
@@ -1,11 +1,12 @@
 [build.environment]
 NODE_VERSION = "16"
+NEXT_USE_NETLIFY_EDGE = "true"
 
 [context.alpha]
 environment = { CHANNEL = "alpha" }
 
 [context.beta]
 environment = { CHANNEL = "beta" }
 
-[dev]
-framework = "#static"
+[[plugins]]
+package = "@netlify/plugin-nextjs"
```

Comment:
No need to add this. It will get auto installed when you deploy.

```suggestion
```

Reply:
So this change will cause the next.js api routes to run as edge functions?

Reply:
Sorry, should have been clearer with this change. This is something Netlify handles for you which is why you don't need it in the configuration. It's not related to anything edge.

Reply:
For the actual issue you're having is it that the og-image isn't rendering or not building during the deploy? I can take a peek at this later tonight.